app/javascript/packages/address-search/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@18f/identity-address-search",
-  "version": "3.1.1",
+  "version": "4.0.0",
   "type": "module",
```

Comment:
According to [Semantic Versioning](https://semver.org/) I could increase this to `3.2.0`.  It looks like the only change to `@18f/identity-address-search` since `3.1.1` is Will's changes in [PR](https://github.com/18F/identity-idp/pull/11224/files). I'd like to error on the cautious side. If you look at [identity-site package.json](https://github.com/GSA-TTS/identity-site/blob/main/package.json), you will see under the dependency list that  `@18f/identity-address-search` uses `^`.  If I version up to `3.2.0` and publish to npm, identity-site will use it. If it is breaking, will break identity-site. I would rather version up in identity-site and have more control in case the change in Will's PR from `import { t } from '@18f/identity-i18n';` to `import { useI18n } from '@18f/identity-react-i18n';` is actually breaking. Initially I thought it was but learning more about package imports in identity-site- I don't believe it is.

Reply:
`identity-site` has a `package-lock.json`, so it won't just automatically upgrade to the latest version satisfying the version range. I think if the changes here meet the criteria of a minor change, we should bump the minor version. The brochure site won't update automatically.

Reply:
If you are confident the brochure sire won't automatically update then I will increase from `3.1.1` to `3.2.0`. I will make the changes now.

Reply:
Yes, it's one of the purposes of a lockfile to pin the site to a specific installed version of the dependency, so that this exact kind of issue doesn't happen where dependencies change unknowingly.

It will only be updated the next time someone explicitly updates the package, either installing the latest version (`npm install @18f/identity-address-search@latest`) or updating the lockfile version within the current version constraint (`npm update @18f/identity-address-search`).